datasets/case01/queries/0006_transientVarObjNearGalaxy.sql

```diff
@@ -0,0 +1,17 @@
+-- Select transient variable objects near a known galaxy
+-- http://dev.lsstcorp.org/trac/wiki/dbQuery015
+
+-- we don't have variability and extendedParamer columns
+
+-- Our case01 dataset is fairly sparse, all objects are futher
+-- apart than is supported by qserv near neighbor queries.
+-- To allow this test to run, object 90030275138483 was duplicated
```

Comment:
I've compared the TSV files of this and the previous version, and found a new row with objectId=1. I presume this is the one you added into the dataset?

Reply:
ya